linkis-engineconn-plugins/engineconn-plugins/elasticsearch/src/main/scala/com/webank/wedatasphere/linkis/engineplugin/elasticsearch/ElasticSearchEngineConnPlugin.scala

```diff
@@ -0,0 +1,57 @@
+package com.webank.wedatasphere.linkis.engineplugin.elasticsearch
```

Comment:
Each class requires a license file. You do not need to specify webank, but only Apache License
Version 2.0
http://www.apache.org/licenses/LICENSE-2.0

Reply:
> Each class requires a license file. You do not need to specify webank, but only Apache License
> Version 2.0
> http://www.apache.org/licenses/LICENSE-2.0

Thanks, I will add the license.